openrarity/collection/collection.py

```diff
@@ -32,27 +32,71 @@
 
 
 class AttributesStatistics(TypedDict):
+    """A class to represent Attribute Statistics."""
```

Comment:
More descriptive. ie: What is an Attribute Statistic

Reply:
fixed it.